### PR TITLE
PAYP-965 UOB Payment

### PR DESCRIPTION
**Pull request by theanh0512**

### CODE REVIEW

File: uikit/src/main/java/com/midtrans/sdk/uikit/views/uob/web/UobWebPaymentActivity.java [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        initActionButton();
+        hideProgressLayout();
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+
+        if (uobEzpayIntentCode == UiKitConstants.INTENT_CODE_UOBEZPAY && presenter != null) {
+            presenter.getPaymentStatus();
```
Comment (askur87) [inline uikit/src/main/java/com/midtrans/sdk/uikit/views/uob/web/UobWebPaymentActivity.java:47]: do we need check status for UOB WEB ?? i think the status will `Pending` right for UOB web and no need to check status

**Comment by askur87** [comment]
Please check the value of `Payment Type`, I think there is still not set with UOB EZ PAY
Screen Shot 2021-06-18 at 9 32 52 PM

File: uikit/src/main/java/com/midtrans/sdk/uikit/views/uob/web/UobWebPaymentActivity.java (hunk 1 above)
Comment (theanh0512) [reply uikit/src/main/java/com/midtrans/sdk/uikit/views/uob/web/UobWebPaymentActivity.java:47]: @askur87 if customer click to pay before returning to the app, we also need to show the  PaymentStatusPage right?
image

Comment (askur87) [reply uikit/src/main/java/com/midtrans/sdk/uikit/views/uob/web/UobWebPaymentActivity.java:47]: hmm... for UOB web I think we don't need to show paymentStatus page just return the status as pending, you can stimulate using e-Pay BRI, the flow should be same

**Review by askur87 — APPROVED** [review]

**Review by uziwuzzy — APPROVED** [review]

**Pull request merged**